src/slic3r/GUI/Tab.cpp

```diff
@@ -3843,25 +3843,25 @@ void TabPrinter::build_fff()
         // optgroup->append_single_option_line("spaghetti_detector");
         optgroup->append_single_option_line("time_cost");
         
-        optgroup  = page->new_optgroup(L("Cooling Fan"), "param_cooling_fan");
+        optgroup  = page->new_optgroup(L("Cooling Fan"), L"param_cooling_fan");
         Line line = Line{ L("Fan speed-up time"), optgroup->get_option("fan_speedup_time").opt.tooltip };
         line.append_option(optgroup->get_option("fan_speedup_time"));
         line.append_option(optgroup->get_option("fan_speedup_overhangs"));
         optgroup->append_line(line);
         optgroup->append_single_option_line("fan_kickstart");
 
-        optgroup = page->new_optgroup(L("Extruder Clearance"), "param_extruder_clearence");
+        optgroup = page->new_optgroup(L("Extruder Clearance"), L"param_extruder_clearence");
```

Comment:
I believe this should be `L"param_extruder_clearance"`, but I am not sure where else this name should be fixed.